spec/features/combined_teacher_claim_journey_with_teacher_id_spec.rb

```diff
@@ -175,11 +175,8 @@
     choose "Male"
     click_on "Continue"
 
-    expect(claim.reload.payroll_gender).to eq("male")
-
-    click_link "Back"
-
-    expect(page).to have_text(I18n.t("questions.payroll_gender"))
+    # - teacher-reference-number removed from user journey
+    expect(page).not_to have_text(I18n.t("questions.teacher_reference_number"))
```

Comment:
It might be more reliable to assert the presence of some known text; there's many pages or scenarios where this will pass unintentionally.

Reply:
Good point @slorek; I've blindly put back the lines that I thought I had accidentally changed during a rebase, as you can see this test is currently behaving like this on [master](https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/master/spec/features/combined_teacher_claim_journey_with_teacher_id_spec.rb#L172-L179)

It looks like during the rebase I was actually trying to address it, so I'll put back my correction with a comment.

Reply:
Addressed with [2fd3812](https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/2653/commits/2fd3812b68b44214055291b8f53f1d66ee4ef006)